.env.example

```
@@ -117,11 +117,15 @@ IVAS_WS_URL=ws://localhost:8000
 IVAS_GEMINI_API_KEY=
 IVAS_GEMINI_LIVE_MODEL=gemini-2.0-flash-live-001
 
-# -----------------------------------------------------------------------------
 # Service Specific: ACAFS (Python)
 # -----------------------------------------------------------------------------
 ACAFS_SVC_PORT=8102
 ACAFS_SVC_DB_NAME=acafs_db
+# Single OpenRouter key powers reasoning (Gemma 4), grading (Qwen3-Coder), and chat (MiniMax)
+ACAFS_OPENROUTER_API_KEY=your-openrouter-api-key-here
+ACAFS_CHAT_MODEL=minimax/minimax-m2.5:free
+ACAFS_REASONER_MODEL=google/gemma-4-26b-a4b-it:free
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACAFS_REASONER_MODEL key should go before the ACAFS_SVC_DB_NAME key

(UnorderedKey)